docs/design/autoware-concepts/difference-from-ai-and-auto.md

```diff
@@ -0,0 +1,22 @@
+
+Note that Autoware.AI is currently in maintenance mode and is planning to finish commitment to the repository after end of 2022.
+
+## Autoware.Auto
+
+[Autoware.Auto](https://gitlab.com/autowarefoundation/autoware.auto/AutowareAuto) is an autonomous driving software based on ROS2. When we did transition to ROS2, we decided not to do simple porting of ROS1 to ROS2 of Autoware.AI. Instead, we went for rewriting the code basically from scratch with proper engineering practices, including defining target use case (AVP, cargo delivery, etc), designing architecture, writing design documents and test codes.
+
+Autoware.Auto seemed to work fine initially, but after going under development of AVP and Cargo Delivery, we started to see some of the following issues:
+
+- The requirement set for merging features into Autoware.Auto is high, and it became difficult for researchers and students to join in to the development. Most of the developers of Autoware.Auto is from the companies who are in AWF organization. As a result, lacked place to have state-of-the-art features from papers.
+- What we also saw was that when we were trying to make a relatively large change to the code in architecture level, it was difficult to do on main branch of Autoware.Auto. To try out experimental architecture, we had very large overhead to keep main branch stable and also making sure every change satisfied the CI.
+
+## Autoware.Core/Universe
+
+In order to address issues we had in Autoware.Auto, we have decided to maintain code base with Autoware Core/Universe architecture. Autoware.Core would carry-over the policy of Autoware.Auto with a stable, well-tested code. In addition to that, we have created new repository Autoware.Universe which would act as an extension of Autoware.Core, where users can choose to replace a component (e.g. localization module) of Core with Universe component if they wish to use experimental features. Universe would have more relaxed code quality requirements compared to Core, but wouldn't be as loose as Autoware.AI. If we find features in Universe to be useful, we could also have AWF engineers porting those codes to be merged into Core as well. This way, we aim to achieve both requirements to have a stable and safe autonomous driving system while enabling access to state-of-the-art features generated by third-party contributors.
```

Comment:
Changed `Autoware.Core` and `Autoware.Universe` to `Autoware Core` and `Autoware Universe` since those are supposed to be the official names of each codebase. If not, then we need to check the documentation site and AWF website for consistency.

I rewrote quite a lot of this section too, so again please check to make sure I have not removed or changed the originally intended meaning.

```suggestion
In order to address the issues with Autoware.Auto development, the Autoware Foundation decided to create a new architecture called Autoware Core/Universe.

Autoware Core carries over the original policy of Autoware.Auto to be a stable and well-tested codebase. Alongside Autoware Core is a new repository called Autoware Universe which acts as an extension of Autoware Core that allows users to replace a component (e.g. the Localization module) of Core with a Universe component in order to use more experimental features. 

Universe has a more relaxed code quality requirements compared to Core, but will not be as loose as that of Autoware.AI. Also, if any features added to Universe are deemed to be useful to the wider community, AWF engineers could potential port that code for merging into Core. 

This way, the primary requirement of having a stable and safe autonomous driving system can be achieved, whilst simultaneously enabling access to state-of-the-art features created by third-party contributors.
```